packages/protocol/src/data/part-data.ts

```diff
@@ -1,78 +1,34 @@
-import {
-  Data,
-  ElementAlternative,
-  ElementEMail,
-  ElementScript,
-  ScriptMapCode,
-  ScriptParameterizedMapCode,
-  StartPermission,
-  StartRequest,
-  WfCase,
-  WfTask
-} from './inscription';
+import { Data } from './inscription';
+import { ConfigData } from './inscription-data';
 
 export type NameData = Omit<Data, 'config'>;
 
-export type ProcessCallData = {
-  processCall: string;
-};
+export type ProcessCallData = Pick<ConfigData, 'processCall'>;
 
-export type DialogCallData = {
-  dialog: string;
-};
+export type DialogCallData = Pick<ConfigData, 'dialog'>;
 
-export type MappingData = ScriptMapCode;
-export type Mapping = MappingData['map'];
+export type CallData = Pick<ConfigData, 'call'>;
 
-export type CallData = {
-  call: MappingData;
-};
+export type EndPageData = Pick<ConfigData, 'page'>;
 
-export interface EndPageData {
-  page: string;
-}
+export type OutputData = Pick<ConfigData, 'output' | 'sudo'>;
 
-export type OutputData = ElementScript;
+export type ConditionData = Pick<ConfigData, 'conditions'>;
 
-export type ConditionData = ElementAlternative;
+export type TaskData = Pick<ConfigData, 'persistOnStart' | 'task' | 'tasks'>;
 
-export interface TaskData {
-  task: WfTask;
-  tasks: WfTask[];
-  persistOnStart: boolean;
-}
```

Comment:
Do no longer define our own types, but extract the needed configs from the ConfigData.
This has the benefit, that e.g. the ts typechecker fails if the `persist` flag was renamed to `persistOnStart`